protocol.html

```diff
@@ -723,27 +723,27 @@ <h4 property="schema:name"><span>Note</span>: Conditional Update</h4>
 
                     <p>An <em>N3 Patch</em> is a document in the <em>Notation3 (N3)</em> format [<cite><a class="bibref" href="#bib-notation3">N3</a></cite>], identified by the media type <code>text/n3</code>, conforming to the following structural constraints:</p>
 
                     <ul>
-                      <li>The patch document can contain patch resources identified by a URI <code>?patch</code> which can be chosen freely.</li>
-                      <li>A patch resource may contain a triple <code>?patch rdf:type solid:Patch</code>.</li>
-                      <li>A patch resource contains at most one triple of the form <code>?patch solid:deletes ?deletions</code>.</li>
-                      <li>A patch resource contains at most one triple of the form <code>?patch solid:inserts ?insertions</code>.</li>
-                      <li>A patch resource contains at most one triple of the form <code>?patch solid:where ?conditions</code>.</li>
-                      <li>When present, <code>?deletions</code>, <code>?insertions</code>, and <code>?conditions</code> must be non-nested <em>cited formulae</em> [<cite><a class="bibref" href="#bib-notation3">N3</a></cite>] consisting only of triples and/or triple patterns. When not present, they are presumed to be the empty formula <code>{}</code>.</li>
+                      <li id="server-patch-n3-identifier" rel="spec:requirement" resource="#server-patch-n3-identifier"><span property="spec:statement">The patch document <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> contain patch resources identified by a URI or blank node <code>?patch</code> which can be chosen freely.</span></li>
+                      <li id="server-patch-n3-type" rel="spec:requirement" resource="#server-patch-n3-identifier">A patch resource <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> contain a triple <code><span property="spec:statement">?patch rdf:type solid:Patch</span></code>.</li>
+                      <li id="server-patch-n3-identifier" rel="spec:requirement" resource="#server-patch-n3-deletes"><span property="spec:statement">A patch resource <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> contain at most one triple of the form <code>?patch solid:deletes ?deletions</code>.</span></li>
+                      <li id="server-patch-n3-inserts" rel="spec:requirement" resource="#server-patch-n3-inserts">A patch resource <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> contain at most one triple of the form <code>?patch solid:inserts ?insertions</code>.</li>
+                      <li id="server-patch-n3-where" rel="spec:requirement" resource="#server-patch-n3-where"><span property="spec:statement">A patch resource <span rel="spec:requirementLevel" resource="spec:MAY">MAY</span> contain at most one triple of the form <code>?patch solid:where ?conditions</code>.</span></li>
```

Comment:
RFC `MAY` does not have the same meaning as appears to be intended by these statements.  `MAY contain at most` does not restrict the resource being described at all.

Reply:
Thanks, clarified in 03d8df7e6d224fd7d3dfaa2e857c9c8bcc15a119